show_route/route/views.py

```
@@ -0,0 +1,308 @@
+import folium
+from django.shortcuts import render,redirect
+
+
+def showmap(request):
+    return render(request,'showmap.html')
+
+import osmnx as ox
+import numpy as np
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)